### ISSUE TITLE
Make code snippets accessible.

### ISSUE THREAD

**Issue by catiarodriguescosta** [wmcadigital/wmn-design-system#279]
**Describe the bug**
Axe audit showed some colour contrast issues and some concerns about keyboard access to scroll the region.

**To Reproduce**
1. Go to 'https://designsystem.wmnetwork.co.uk/styles/icons/'
2. Scroll down to the first snippet
3. See error

**Expected behavior**
Code snippets should have a good colour contrast and should be accessible.
